modules/i18n/src/translations/fr-FR/portals/console.ts

```diff
@@ -8905,6 +8905,11 @@ export const console: ConsoleNS = {
                             header: "Êtes-vous sûr ?",
                             message: "Cette action est irréversible et supprimera définitivement le {{type}} " +
                                 "sélectionné"
+                        },
+                        deleteItemError: {
+                            content: "Remove the associations from following application before deleting:",
+                            header: "Unable to Delete",
+                            message: "There is an application using this role."
```

Comment:
Let's add the proper FR translations

Reply:
Fixed: 4d3576dd5018d4d3f61232c637f00d45eb995d1d